app/src/main/java/protect/card_locker/ScanActivity.java

```diff
@@ -47,6 +57,8 @@ public class ScanActivity extends CatimaAppCompatActivity {
     private String addGroup;
     private boolean torch = false;
 
+    private Toast camPermissionToast;
```

Comment:
I don't think we need a toast because the text on the screen is already very clear. I'd rather keep the code simpler when it doesn't add much.

Reply:
Sure, that sounds very reasonable. I will remove this redundancy.